modules/oadp-multiple-bsl-use-case.adoc

```diff
@@ -0,0 +1,177 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/configuring-oadp-multiple-bsl.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-multiple-bsl-use-case_{context}"]
+= {oadp-short} use case for two BSLs
+
+In this use case, you configure the DPA with two storage locations by using two cloud credentials. You backup a `MySQL` database application by using the default BSL. {oadp-short} stores the backup resources in the default BSL. You then backup the `MySQL` application again by using the second BSL.
```

Comment:
I think we are missing "will"s here

example: "In this use case, you **will** configure the DPA with..."

Reply:
The [Red Hat style guide](https://redhat-documentation.github.io/supplementary-style-guide/#_w) for documentation specifies that we avoid the usage of 'will'. We are supposed to write in first person, present tense :).